README.md

```diff
@@ -1,8 +1,18 @@
 # Synergy Design System
-<img src="./packages/docs/public/synergy-logo-light.png" alt="Synergy-Logo" width="400"/><br>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./packages/docs/public/synergy_logo_dark.png" width="400">
+  <img alt="Logo" src="./packages/docs/public/synergy_logo_light.png" width="400">
+</picture>
+
 Welcome to the home of the Synergy Design System.
 This project is mainly intended for usage in [SICK](https://www.sick.com) applications to ensure a unique look and feel across applications, however you may also use it [for your own projects for free](./LICENSE) as well.
 
+## Documentation
+The documentation is always up-to-date and can found on https://synergy-design-system.github.io/ .
```

Comment:
```suggestion
The documentation is always up-to-date and can be found on https://synergy-design-system.github.io/ .
```

Reply:
Please update all `homepage` values in all `package.json` (e. g. https://github.com/synergy-design-system/synergy-design-system/blob/2bc257dd5c35ddf362239e21dc9bfc4f7629faa4/packages/angular/package.json#L7) as well, so that they are linked to Storybook.

And update the READMEs as well to list Source and Documentation, that Storybook is reflected there, too, as this is the only thing people see on NPM.

There I would just do:

```
- Documentation: https://synergy-design-system.github.io/
- Source Code: https://github.com/synergy-design-system/synergy-design-system
```

The change only in main README doesn't provide much value, as this is only visible on GitHub, neither on NPM or Storybook, so linking the Source doesn't provide any value at all.